### PR TITLE
bloc115 -- create unit tests for account data service (dbs_account)

### PR DESCRIPTION
**Pull request by alexplevako**

### CODE REVIEW

File: tests/tests/account_data_service_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+{
+    try
+    {
+        create_account();
+
+        const account_object& acount = db.get_account("user");
+
+        BOOST_CHECK(acount.name == "user");
+    }
+    FC_LOG_AND_RETHROW()
```
Comment (abolinbsu) [inline tests/tests/account_data_service_tests.cpp]: Why not just FC_CAPTURE_AND_LOG

Comment (alexplevako) [reply tests/tests/account_data_service_tests.cpp]: if there is exception test should fail, that's why we do rethrow

Comment (alexplevako) [reply tests/tests/account_data_service_tests.cpp]: I found only one usage of FC_CAPTURE_AND_LOG in the whole project.

Comment (alexplevako) [reply tests/tests/account_data_service_tests.cpp]: with FC_CAPTURE_AND_LOG output would be:
*** No errors detected

**Pull request merged**